src/r5py/r5/__init__.py

```diff
@@ -2,7 +2,8 @@
 
 """R5 classes."""
 
-from .. import util  # noqa: F401
+# start the JVM first thing
+from ..util import jvm  # noqa: F401
```

Comment:
These are good, but spotted that the import for `SpeedUnit` is missing and I guess the class does not exist yet in the project?

I.e. [this line](https://github.com/r5py/r5py/blob/585947f56614964089d2cf6bd89ede804d40f247/src/r5py/r5/__init__.py#L22)

Reply:
True! Just have to get off my train, but I’ll check as soon as I have a chance.

Reply:
I remember that I had foreseen a `SpeedUnit` enum at some point, but then decided against it (not enough benefits for the maintenance effort). Clearing that out, I seem to have remove the wrong import here: it should have been `SpeedConfig`